lib/ioutil/log_entry.go

```diff
@@ -18,12 +18,9 @@ import (
 	"time"
 )
 
-const logEntryVersion = 1
-
 // LogEntry attaches a timestamp to a log entry
 // easyjson:json
 type LogEntry struct {
-	Log     string    `json:"log"`
-	Time    time.Time `json:"time"`
-	Version int       `json:"version"`
+	Msg string    `json:"msg"`
+	Ts  time.Time `json:"ts"`
```

Comment:
👍 I will change to int64 once I understand how we will need to use it with `docker logs --since`.